rules/windows/collection_posh_audio_capture.toml

```diff
@@ -49,10 +49,17 @@ authorized benign true positives (B-TPs), exceptions can be added.
 ### Response and remediation
 
 - Initiate the incident response process based on the outcome of the triage.
+- Isolate the involved hosts to prevent further post-compromise behavior.
 - The response must be prioritized if this alert involves key executives or potentially valuable targets for espionage.
-- Quarantine the involved host for forensic investigation, as well as eradication and recovery activities.
-- Configure AppLocker or equivalent software to restrict access to PowerShell for regular users.
-- Review GPOs to add additional restrictions for PowerShell usage by users.
+- Investigate credential exposure on systems compromised or used by the attacker to ensure all compromised accounts are
+identified. Reset passwords for these accounts and other potentially compromised credentials, such as email, business
+systems, and web services.
+- Restrict PowerShell usage outside of IT and engineering business units using GPOs, AppLocker, Intune, or similar software.
+- Run a full antimalware scan. This may reveal additional artifacts left in the system, persistence mechanisms, and
+malware components.
+- Determine the initial vector abused by the attacker and take action to prevent reinfection via the same vector.
```

Comment:
```suggestion
- Determine the initial vector abused by the attacker and take action to prevent reinfection through the same vector.
```

Reply:
We usually try to avoid "via" for accomodate for ESL users. If you accept this change, you can do a find and replace all to quickly implement it.